docs/tutorial/helloworld.py

```diff
@@ -0,0 +1,29 @@
+import time
+import http.server
+import os
+
+
+HOST_NAME = '0.0.0.0' # Host name of the http server
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+
+class MyHandler(http.server.BaseHTTPRequestHandler):
+    def do_GET(s):
+        """Respond to a GET request."""
+        s.send_response(200)
+        s.send_header("Content-type", "text/html")
+        s.end_headers()
+        s.wfile.write("<html><head><title>Time Server</title></head>")
+        s.wfile.write("<body><p>The current time is %s</p>" % time.asctime())
```

Comment:
Python 3 convention is to use [`str.format()`](https://docs.python.org/3.1/library/stdtypes.html#str.format):

```python
s.wfile.write("<body><p>The current time is {}</p>".format(time.asctime()))
```